source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRContentStackView.mm

```diff
@@ -557,20 +557,37 @@ - (CGFloat)getViewWithMaxDimensionAfterExcluding:(UIView *)view dimension:(CGFlo
     return currentBest;
 }
 
+- (NSString *)gatherAccessibilityLabelsFromSubviews
+{
+    return nil;
+}
+
 - (void)configureForSelectAction:(ACOBaseActionElement *)action rootView:(ACRView *)rootView
 {
     if (action != nullptr) {
-        NSObject<ACRSelectActionDelegate> *target = nil;
+        ACRBaseTarget *target = nil;
         if (ACRRenderingStatus::ACROk == buildTarget([rootView getSelectActionsTargetBuilderDirector], action, &target)) {
             [self addTarget:target];
             self.selectActionTarget = target;
             setAccessibilityTrait(self, action);
+            self.accessibilityLabel = configureForAccessibilityLabel(action, [self gatherAccessibilityLabelsFromSubviews]);
```

Comment:
why not setting nil in here? is this method overriden by other classes?

Reply:
Good question. iOS doesn't support synthesizing accessibility labels from its content views or subviews.
That function will be implemented in our SDK in the future and the synthesized label will be accessed through the method that just returns nil.